### DISCUSSION TITLE
Number of deviants and number of cnvs

### DISCUSSION THREAD

**Discussion by Helithia** [piyalkarum/rCNV#3] [Q&A]
Hello, 

I am currently using your package on GBS data (reference based). I’m trying to clean my dataset of all cnvs to only keep non duplicated SNPs. To do so, I applied the function dupGet with z.05 and chi.05 statistics and found 782 deviants. Then I applied the function cnv with the same statistics and found 6641 cnvs. In the article you published last year, the number of deviants is always greater than the number of cnvs, which is not my case here. Is it normal to have more cnvs than deviants ? If not, do you think that you could help me ?

Sincerely, 

Nolwenn Béchennec

**Comment by piyalkarum** [comment]
Hi Nolwenn,

Thanks for using our package.
Generally, you shouldn't get more CNVs than deviants but sometimes you can get more CNVs depending on the filtering method you use. I would recommend you use the "intersection" option. "kmeans" is a bit less stringent (it also depends on your data). 

Also from the difference of the deviants and cnvs in your two analyses, I would say that the stat types you're using might not be the best for your data. I suggest you check the Z and chi square value plots and decide which stat fits your data best. Another thing to consider is if your method of sequencing can have probe bias. See our tutorial on this. 

Hope this helps.
Feel free to reach out if you need more help.

Best of luck
